javascript/javascript3/week2/fetchWait.js

```diff
@@ -0,0 +1,30 @@
+//promise
+function fetchWait(delayTime) {
```

Comment:
Try and seperate the call to the api from the timer logic. The timer logic should be a function that takes a delay time and returns a promise. Inside it should resolve when the setTimeout callback is called